README.md

```diff
@@ -126,6 +126,12 @@ Below you will find descriptions of all rules available in **AEM Rules for Sonar
 - **AEM-16** Optional is defined as ``DefaultInjectionStrategy``
   - Usage of ``@Optional`` annotation is redundant, when ``defaultInjectionStrategy`` is ``OPTIONAL``.
 
+## [HTL](https://github.com/Adobe-Marketing-Cloud/htl-spec/blob/master/SPECIFICATION.md) related
```

Comment:
I'd put this section as the first one to promote new functionality.

Reply:
moved